apps/portal/redirects.mjs

```diff
@@ -1017,6 +1015,8 @@ const otherRedirects = {
   "/wallet/get-started/overview": "/connect/wallet/overview",
   // connect > wallets
   "/connect": "/wallets",
+  "/connect/auth": "/wallets/auth",
+  "/connect/:path*": "/wallets/:path*",
```

Comment:
Can we also go through and change all the links to new ones so there's no extra redirect and we make sure those links work